src/tek-poe.h

```diff
@@ -15,6 +15,9 @@
 
 struct port_state {
 	const char *status;
+	float mvolt;
+	float mampere;
```

Comment:
Just use SI units here  -- `voltage` and `current` for variable names with units [V], and [A] respectively. For someone without scientific training, it may not be obvious if it's micro, milli, or mega.

Reply:
Well, I chose this because the protocol returns mV and mA, and I wanted to stay close to the unit the protocol documents.
Nevertheless I can normalise it to volts and amperes too.

Reply:
Alternatively, you could use `voltage_mv` and `current_ma`.

Reply:
I like that suggestion

Reply:
No. Use SI units, no prefix.

Reply:
Don't see what this has to do with it.  This struct stores a value reading.  The way in which we present the value is completely unrelated, and handled in the formatting.  Internal value following the hardware/remote unit vs exposed value which is conforming standards, previous behaviours, etc.

Reply:
I've re-read the actual change and, yes, I see your point; still, the `port_state` struct is used in a few places, and doesn't elsewhere reflect the protocol, but instead a post-processed view atop of it (e.g. `port_state.poe_mode`).